modules/tasks/tasks/src/main/java/org/motechproject/tasks/web/ChannelController.java

```diff
@@ -98,9 +101,18 @@ public TriggersLists getTriggers(@RequestParam String moduleName,
      * @param info  the information about the trigger
      * @return the trigger matching given information
      */
-    @RequestMapping(value = "channel/trigger", method = RequestMethod.POST)
+    @RequestMapping(value = "channel/trigger", method = RequestMethod.GET)
     @ResponseBody
-    public TriggerEvent getTrigger(@RequestBody TaskTriggerInformation info) {
+    public TriggerEvent getTrigger(TaskTriggerInformation info) {
```

Comment:
Does this refactor even make sense? If we have got to provide a huge object as URL parameter, it would probably be better to leave it as POST payload.